static/app/views/starfish/views/spanSummaryPage/sampleList/index.tsx

```diff
@@ -14,6 +15,7 @@ type Props = {
 
 export function SampleList({groupId, transactionName, transactionMethod}: Props) {
   const router = useRouter();
+  const [highlightedSpanId, highlightSample] = useState<string | undefined>(undefined);
```

Comment:
This name mismatch is a little weird!
```suggestion
  const [highlightedSpanId, setHighlightedSpanId] = useState<string | undefined>(undefined);
```

would make more sense IMO